app/src/main/res/layout/user_grid.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_marginEnd="@dimen/margin_6dp"
+    android:layout_marginBottom="@dimen/margin_12dp"
+    android:layout_marginStart="@dimen/margin_6dp"
+    android:layout_marginTop="@dimen/margin_6dp"
```

Comment:
margin horizontal can be used to replace start and end margin since they are the same